Content.Server/Nutrition/EntitySystems/DrinkSystem.cs

```diff
@@ -165,6 +166,9 @@ private bool TryDrink(EntityUid user, EntityUid target, DrinkComponent drink, En
         if (!HasComp<BodyComponent>(target))
             return false;
 
+        if (HasComp<BorgChassisComponent>(target))
+            return false;
```

Comment:
```c#
 if (!HasComp<BodyComponent>(target) || (HasComp<BorgChassisComponent>(target))
            return false;
```

Reply:
I did it that way for readability reasons over code conciseness, but it's a simple change if needed.

Reply:
It is still readable. Take a look at Content.Client/Doors/AirlockSystem.cs, line 93 for example

Reply:
I don't think cyborgs should be hardcoded into the DrinkSystem. How does the FoodSystem handle this?

Reply:
> I don't think cyborgs should be hardcoded into the DrinkSystem. How does the FoodSystem handle this?

The food system checks for an `UnremoveableComponent` and returns out of the `TryFeed` method if it is present. All items given to borgs via `ItemBorgModuleComponent` are given `UnremoveableComponent`, which prevents them from eating. `DrinkSystem` has no such check, should I use that approach instead?

Reply:
Is also worth mentioning that the `FoodSystem` checks for a stomach component in the `TryFeed` method, which borgs do not have, while the `DrinkSystem` does not. If an entity gets past all the checks in `TryDrink`, then it assumes it has a stomach to put the drink into, which is why the solution is deleted from existence in the case of borgs.